typescript/infra/config/environments/test/hooks.ts

```diff
@@ -0,0 +1,36 @@
+import { ChainMap, objMap } from '@hyperlane-xyz/sdk';
+import {
+  HookConfig,
+  MessageHookConfig,
+  NativeType,
+  NoMetadataIsmConfig,
+} from '@hyperlane-xyz/sdk/dist/hook/types';
+import { types } from '@hyperlane-xyz/utils';
+
+import { owners } from './owners';
+
+const filteredOwners: ChainMap<types.Address> = Object.keys(owners).reduce(
```

Comment:
Consider making a reusable filter function something in the `utils` folder (like `objects.ts`)` instead of redefining this filter function in >1 place.
If you decide to keep it, please apply the same feedback as above